test.txt

```diff
@@ -1 +1 @@
-test
\ No newline at end of file
+Hello world
```

Comment:
Great change, but I think it would be better to say "Hi world"

Reply:
Thank you, I've changed it.